packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.tsx

```diff
@@ -21,7 +21,7 @@ export interface IDetailsColumnProps extends React.Props<DetailsColumn> {
   onColumnContextMenu?: (column: IColumn, ev: React.MouseEvent<HTMLElement>) => void;
   dragDropHelper?: IDragDropHelper | null;
   isDraggable?: boolean;
-  setDraggedItemIndex?: (itemIndex: number) => void;
+  updateDragInfo?: (itemIndex: number, event?: MouseEvent) => void;
```

Comment:
All function callbacks for eventing should use a `props` object and make the `event` required, e.g.:
```ts
updateDragInfo(props: { itemIndex: number; }, event: MouseEvent);
```
Also, nitpick that all these callbacks should have `props, event` in the same order.